Formula/dotnet@6.rb

```diff
@@ -0,0 +1,236 @@
+class DotnetAT6 < Formula
+  desc ".NET Core"
+  homepage "https://dotnet.microsoft.com/en-us/"
```

Comment:
Used US site as `brew audit --new` complained about https://dotnet.microsoft.com/ redirecting to HTTP.